### PR TITLE
Add command to extend cluster expiration timestamp

### LINKED ISSUE #1: add command for extending the expiration date of a cluster
Having a command to easily extend the expiration date of a cluster would be convenient :cat: 

`extend-expiration --cluster-id <CLUSTER_ID> 2h`
`extend-expiration --cluster-id <CLUSTER_ID> 1d`
`extend-expiration --cluster-id <CLUSTER_ID> 20m`

### PR DESCRIPTION
**Pull request by ryankwilliams**
PR adds a new command for extending (setting) the cluster expiration timestamp.

```shell
./out/ocm-toolbox set-cluster-expiration --cluster-id <CLUSTER_ID> --duration 120 --url staging
Original expiration timestamp: 2022-11-22 23:00:00 +0000 UTC
Updated expiration timestamp: 2022-11-23 01:00:00 +0000 UTC
```

Closes #1

### CODE REVIEW

File: pkg/clusterexpiration/clusterexpiration.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+		ExpirationTimestamp(clusterDetails.ExpirationTimestamp().Add(time.Duration(duration) * time.Minute)).
+		Build()
+	if err != nil {
+		fmt.Printf("Unable to patch cluster expiration timestamp: %s, error: %s", clusterID, err)
+		os.Exit(1)
+	}
+
+	_, err = cluster.Update().Body(patchedCluster).Send()
+	if err != nil {
+		fmt.Printf("Failed to update cluster expiration timestamp: %s, error: %s", clusterID, err)
```
Comment (jbpratt) [inline pkg/clusterexpiration/clusterexpiration.go:30]: exit on err?

File: pkg/clusterexpiration/clusterexpiration.go [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+	}
+
+	fmt.Printf("Updated expiration timestamp: %s", patchedCluster.ExpirationTimestamp())
```
Comment (jbpratt) [inline pkg/clusterexpiration/clusterexpiration.go:33]:
```suggestion
	fmt.Printf("Updated expiration timestamp: %s\n", patchedCluster.ExpirationTimestamp())
```

add newline to end of printf

File: pkg/clusterexpiration/clusterexpiration.go (hunk 1 above)
Comment (ryankwilliams) [reply pkg/clusterexpiration/clusterexpiration.go:30]: Thanks for catching this!

File: pkg/clusterexpiration/clusterexpiration.go (hunk 2 above)
Comment (ryankwilliams) [reply pkg/clusterexpiration/clusterexpiration.go]: Thanks for catching this!

**Pull request merged**